packages/compass-components/src/components/leafygreen.tsx

```diff
@@ -26,6 +26,12 @@ import {
   MongoDBLogo,
 } from '@leafygreen-ui/logo';
 import { Menu, MenuSeparator, MenuItem } from '@leafygreen-ui/menu';
+
```

Comment:
Can we add a comment with the description to help future readers?
```suggestion

// If a leafygreen Menu (and therefore MenuItems) makes its way into a <form>,
// clicking on a menu item will submit that form. This is because it uses a button
// tag without specifying a type and buttons by default have type="submit".
```